lib/webpacker/configuration.rb

```diff
@@ -56,6 +56,16 @@ def data
 
     def load
       YAML.load(config_path.read)[env].deep_symbolize_keys
+
+    rescue Errno::ENOENT => e
+      raise "Webpacker configuration file not found #{config_path}. " \
+            "Have you installed webpacker? " \
```

Comment:
I think this line should be replaced with `Please run: rails webpacker:install`. What do you think?